Adds a basic REST endpoint and basic React Dashboard UI

thatonejason opened the pull request.

whitecl commented:
I wonder if a more RESTful approach to the API controller would make sense here, perhaps something like:

/api/v1/responses?recent=true

for the request endpoint

thatonejason commented:
> I wonder if a more RESTful approach to the API controller would make sense here, perhaps something like:
> 
> /api/v1/responses?recent=true
> 
> for the request endpoint

I changed the resource to just be all responses and moved it to its own controller rather than Dashboard. I think the filtering can be part of an exercise for now. What do you think @whitecl?

whitecl reviewed (approved):
👍
